project/Game.py

```diff
@@ -156,6 +180,40 @@ def create_game(self, *args, **kwargs):
         return self.maker(*args, **kwargs)
 
 
+class TestSetPenaltyValue(unittest.TestCase):
+    def setUp(self):
+        self.game = GameFactory(make_game).create_game()
+        self.game.started = False
+
+    def test_set_penalty_positive(self):
+        pointValue = 10
+        self.assertTrue(self.game.set_point_penalty(pointValue), "Point value not setting correctly")
```

Comment:
in addition to testing the return value, check that the data member was actually changed.